### ISSUE TITLE
"Let a thousand flowers bloom"

### ISSUE THREAD

**Issue by matkoniecz** [pietervdvn/MapComplete#317]
Is it really intentional?

1) It is a bit of mistranslation ( https://en.wiktionary.org/wiki/let_a_thousand_flowers_bloom )

2) given context of that ( https://en.wikipedia.org/wiki/Hundred_Flowers_Campaign ) it is weird to see it used in anything intended as positive

> After this brief period of liberalization, the crackdown continued through 1957–1959 as an Anti-Rightist Campaign against those who were critical of the regime and its ideology. Citizens were rounded up in waves by the hundred of thousands, publicly criticized, and condemned to prison camps for re-education through labor, or even execution.[5] The ideological crackdown re-imposed Maoist orthodoxy in public expression, and catalyzed the Anti-Rightist Movement. 

On top of that it is likely that this was malicious deception from start (see https://en.wikipedia.org/wiki/Hundred_Flowers_Campaign#Debated_intention_of_the_campaign )

**Comment by joostschouppe** [comment]
I can't read Pieter's mind, but:
* I read it more in a "flower power" kind of way, like in this quote from 1967: "The cry of 'Flower Power' echoes through the land. We shall not wilt. Let a thousand flowers bloom."
* From wiktionary: "As some usage shows, those knowing what happened to those who took Mao's encouragement of 1956 seriously view uses of this statement skeptically and may use it ironically."
For me this reads a little "tongue in cheek":
- the idea of helping people grow flowers in the thousands is actually a good fit to the project
- the fact that there is this dark association highlights the risks involved with a project like MapComplete, willingness to deal with those risks, but also a kind of light hearted approach to those issue. "Let's just fix the problems when we cause them, not sit quietly in a corner in fear of making mistakes"

The fact that this does "work" for me, might be because I'm from very similar background as Pïeter. For example, there was a slightly left wing weekly for a while in Flanders called "Mao magazine". I think the name was chosen as a way to intentially annoy right wing opponents (who like to find links between left wing activists and communist likings) and actively deal with earlier left wing activists that actually did seriously and naively endorse Mao. Then, when that magazine went bust, they followed it up with Deng - which many people thought hilarious.

Now I can imagine that if the political history here hits closer home, this kind of joking is perhaps absolutely not funny.

**Comment by matkoniecz** [comment]
In general: it is not something very problematic, if used intentionally feel free to ignore this issue and close it.

> Now I can imagine that if the political history here hits closer home, this kind of joking is perhaps absolutely not funny.

Yeah, I am from Poland and communist-approval-adjacent jokes are not particularly funny to me ( https://en.wikipedia.org/wiki/Category:Stalinism_in_Poland + family stories of people murdered + communism devastation of country still has impact here).

> "Mao magazine". I think the name was chosen as a way to intentially annoy right wing opponents (who like to find links between left wing activists and communist likings) 

yeah, for me "Mao magazine" is about as funny "Hitler magazine"  as a way to intentionally annoy opponents and make fun of possible association with Nazism

To be clear - "Let a thousand flowers bloom" is just a bit of "WAT?" to me and I was unsure whether it is intentionally used with knowledge of associations. Not problematic in way that "Mao magazine" or "Hitler magazine" would be to me.

**Comment by pietervdvn** [comment]
Hi everyone,

I never intended it to be connected to any of this mao/communism stuff - I was not aware of any such link. (I was vaguely aware of hundred flowers campaign and definitively not under that name).

The flowers are simply a metaphor for a MapComplete theme, the goal is to have thousands of themes over time, thus 'thousands of flowers' (with contributors being the bees, by enjoying the flower and pollinating it with more data). The metaphor is somewhat fitting, as everyone (with enough experience) is free to plant their own (to create their own theme). Some of them might die off quickly, others might become popular; at the same time I do not think it is my job to interfere (too much) with those unofficial themes (as long as they do not hurt OSM).

In summary: I did not intend this to be a communist reference and I apologize if this brought some uncomfortable feelings.

Would you have a similar metaphor (without such connotation) that could be fitting? Or do you think that it is fine to keep it as is, given this explanation?

**Comment by pietervdvn** [comment]
Cloing because stale and non-actionable
